src/main/java/org/jitsi/jicofo/bridge/BridgeSelector.java

```diff
@@ -382,13 +382,6 @@ private void notifyBridgeDown(Bridge bridge)
         eventAdmin.postEvent(BridgeEvent.createBridgeDown(bridge.getJid()));
     }
 
-    private void notifyBridgeOffline(Bridge bridge)
```

Comment:
Why don't we need this event anymore?

Reply:
Because the only place that was using it is `JvbDoctor`, which now gets notified about it via a direct API call.